src/components/CardSelect/components/CardInput.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from 'react'
+import { InputCustomProps } from '../../Input/types'
+import { Input } from '../../Input'
+
+interface TCardInput {
+  inputProps?: InputCustomProps
+  disabled?: boolean
+  className?: string
+}
+export const CardInput = (props: TCardInput): ReactElement => {
+  const { inputProps, disabled, className } = props
+  return (
+    <div className={className}>
+      {inputProps ? <Input {...inputProps} className={'mt-16'} disabled={disabled} /> : null}
+    </div>
```

Comment:
in case if you don't have input props you'll have empty div here? So if you don't wont if you can do
```suggestion
   <>{inputProps ?  <div className={className}>
      <Input {...inputProps} className={'mt-16'} disabled={disabled} /> 
    </div> : null}</>
```